test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
+"""
+import logging
+import sys
+
+import pytest
+
+sys.path.insert(0, sys.path[0] + "/test/cql-pytest")
+import nodetool
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, TextType, Column
+from test.pylib.util import unique_name
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch"}
+
+
+# Checks a cluster boot/operations in multi-dc environment
```

Comment:
This comment doesn't really explain what this test is about. First of all, it's not just a "mult-dc environment", it's a particular setup where, if I understand correctly, you have 30 nodes in 30 different DCs.
Second, what does this test actually check? That the 30 nodes are able to boot (how is boot different in the case of 30 different CDs and one DC)? That you are able to create a table in a 30-DC setup? I don't see any data operations below, only the table creation, so maybe you can explain why it's interesting.

Reply:
yes, the idea is to explore a rare dimension - 30-dc setup . If there is any quadratic or cubic logic in scylla from the number of DCs the test should uncover that. calling random_tables() would cover for DML and DDL in such a cluster. I don't know the actual run time of this test now, if it's too big, we could perhaps reduce the number of DCs.